src/jlapi.c

```diff
@@ -352,6 +352,30 @@ JL_DLLEXPORT void (jl_cpu_wake)(void)
     jl_cpu_wake();
 }
 
+#ifndef _OS_WINDOWS_
+#  define UNW_LOCAL_ONLY
+#  include <libunwind.h>
+#define STRINGIFY1(x) #x
+#define STRINGIFY2(x) STRINGIFY1(x)
+asm("_jl_unw_getcontext: movq " STRINGIFY2(unw_tdep_getcontext) "@GOTPCREL(%rip), %rax\n"
```

Comment:
I'm afraid this is not going to work on ARM and power (and we unwinding works at least on aarch64). Maybe use sth like,

```c
JL_DLLEXPORT void (*jl_get_unw_getcontext(void))(unw_context_t*)
{
    return &unw_tdep_getcontext;
}
```
?